### PR TITLE
DEV10128 - update city (TH) and postal code (HK)

### PR DESCRIPTION
**Pull request by lusyar**

### CODE REVIEW

File: Production/master_supplier.sql [hunk 1]
```diff
@@ -351,7 +351,9 @@ left join unnest(product_subcategory) as ps
     , string_agg(distinct hotel_account_number) as Supplier_Bank_Account_Number
     , string_agg(distinct hotel_account_holder_name) as Supplier_Bank_Account_Name
     , string_agg(distinct hotel_swift_code) as Supplier_Bank_BIC_SWIFT_Code
-    , string_agg(distinct coalesce(hotel_postal_code, 'No Postal Code')) as Address_Postal_Code
+    , case 
+        when string_agg(distinct htls.hotel_country_hb) = 'HK' then ''
```
Comment (Rizki-H) [inline Production/master_supplier.sql:355]: Can we add comment why HK is equal to empty?

File: Production/master_supplier.sql [hunk 2]
```diff
@@ -437,6 +439,7 @@ left join unnest(product_subcategory) as ps
     , '' Address_City_Subdivision_2
     , '' Address_City_Subdivision_1
     , case
+        when oth.country_code = 'TH' then ''
```
Comment (Rizki-H) [inline Production/master_supplier.sql:442]: Can we add comment why TH is equal to empty?

File: Production/master_supplier.sql (hunk 1 above)
Comment (anggianggara) [reply Production/master_supplier.sql]: let me add a comment ya

**Review by anggianggara — APPROVED** [review]

**Comment by anggianggara** [comment]
reviewed & approved

**Pull request merged**